app/assets/stylesheets/components/common/_footer_secondary.scss

```diff
@@ -104,14 +104,20 @@
   }
 }
 
-.footer-secondary__sfgb-text {
+.footer-secondary__mps-text {
   display: inline-block;
+}
+.footer-secondary__mps-text--en {
+  width: 98px;
```

Comment:
I'm not sure about giving this an explicit width because it relies too much on the text displaying pretty much the same across browsers and devices. Would it be better to follow the current markup that divides the text into three sections? (P.S. I don't like that either so there doesn't seem to be a prefect solution).

Reply:
So the reason I've done it this way is because it has to break differently on the welsh to the english because they're different lengths. Lets chat in person about this, I'm also not sure about it

Reply:
@davidtrussler I've committed the changes we discussed. Will update the PR description now to explain